src/main/resources/logstash-diagnostic-templates/flow_metrics.html.ftlh

```diff
@@ -14,7 +14,7 @@
             src="https://unpkg.com/react@18.2.0/umd/react.production.min.js"></script>
     <script src="https://unpkg.com/react-dom@18.2.0/umd/react-dom.production.min.js"></script>
     <script crossorigin="anonymous"
-            src="https://unpkg.com/@mui/material@5.13.7/umd/material-ui.production.min.js"></script>
+            src="https://cdn.jsdelivr.net/npm/@mui/material@5.13.7/umd/material-ui.development.js"></script>
```

Comment:
We have to use a non-minified, develoment library? Seems like an odd issue / fix.

Reply:
Right! We can use minified production version, which is 3-times smaller.

Reply:
🤦